code/game/sound.dm

```diff
@@ -33,6 +33,11 @@
 	S.wait = 0 //No queue
 	S.channel = channel || open_sound_channel()
 	S.volume = vol
+	S.environment = 10 //apparently you need to set an env for echo to work. dumb.
+	S.echo = ECHO_GENERIC //default echo, pretty much does nothing
+
+	var/debug_prints = TRUE //just for while this is WIP, it will be removed
```

Comment:
wats testing()

why am i expected to know all these procs and vars that nobody documents or tells you about